js/app.js

```diff
@@ -12,4 +15,40 @@ export default {
   getSocketUrl() {
     return 'ws://localhost:8080/ws';
   },
+
+  followUnfollow(guid, type = 'follow') {
+    if (typeof guid !== 'string' || !guid.toLowerCase().startsWith('qm')) {
+      throw new Error('You must provide a valid guid.');
+    }
+
+    if (guid === this.profile.id) {
+      throw new Error('You can not follow or unfollow your own guid');
+    }
+
+    return $.ajax({
+      type: 'POST',
+      url: this.getServerUrl(`ob/${type}`),
+      data: JSON.stringify({ id: guid }),
+      dataType: 'json',
+    })
+    .done(() => {
+      // if the call succeeds, add or remove the guid from the collection
+      if (type === 'follow') {
+        this.ownFollowing.add({ guid });
+      } else {
+        this.ownFollowing.remove(guid); // remove via id
+      }
+    })
+    .fail((data) => {
+      const followFailedDialog = new Dialog({   // eslint-disable-line no-unused-vars
```

Comment:
if you just remove the variable assignment (`const followFailedDialog`), you won't need the eslint override.